### PR TITLE
more things

### PR DESCRIPTION
**Pull request by amanda-elliott**

### CODE REVIEW

File: fork.txt [hunk 1]
```diff
@@ -5,4 +5,7 @@ This can OH NOW bar maybe?
 
 hiya!
 
-this is new for a branch change
\ No newline at end of file
+this is new for a branch change
```
Comment (njperry) [inline fork.txt:8]: these 2 lines are duplicates

Comment (amanda-elliott) [reply fork.txt:8]: okay, go ahead and fix it

Comment (njperry) [reply fork.txt:8]: okies

**Comment by njperry** [comment]
resolved conflict of duplicate lines

**Pull request merged**